schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -162,10 +163,55 @@ protected Map<String, Object> getSchemaJsonMap() {
                 .build());
   }
 
-  protected void writeSchemaFile(Map<String, Object> schemaJsonMap) throws IOException {
+  protected Map<String, Object> getAlteredSchemaJsonMap() {
+    return ImmutableMap.of(
+        namespace1 + "." + TABLE_1,
+        ImmutableMap.<String, Object>builder()
+            .put("transaction", true)
+            .put("partition-key", Collections.singletonList("pk1"))
+            .put("clustering-key", Arrays.asList("ck1 DESC", "ck2 ASC"))
+            .put(
+                "columns",
+                ImmutableMap.<String, Object>builder()
+                    .put("pk1", "INT")
+                    .put("ck1", "INT")
+                    .put("ck2", "TEXT")
+                    .put("col1", "INT")
+                    .put("col2", "BIGINT")
+                    .put("col3", "FLOAT")
+                    .put("col4", "DOUBLE")
+                    .put("col5", "TEXT")
+                    .put("col6", "BLOB")
+                    .put("col7", "BOOLEAN")
+                    .put("col8", "TEXT")
+                    .put("col9", "BLOB")
+                    .build())
+            .put("secondary-index", Arrays.asList("col3", "col8"))
+            .put("compaction-strategy", "LCS")
+            .put("network-strategy", "NetworkTopologyStrategy")
+            .put("replication-factor", "1")
+            .put("ru", 5000)
+            .build(),
+        namespace2 + "." + TABLE_2,
+        ImmutableMap.<String, Object>builder()
+            .put("transaction", false)
+            .put("partition-key", Collections.singletonList("pk1"))
+            .put("clustering-key", Collections.singletonList("ck1"))
+            .put(
+                "columns",
+                ImmutableMap.of(
+                    "pk1", "INT", "ck1", "INT", "col1", "INT", "col2", "BIGINT", "col3", "FLOAT",
+                    "col4", "TEXT"))
+            .put("network-strategy", "NetworkTopologyStrategy")
+            .put("replication-factor", "1")
+            .build());
+  }
+
+  protected void writeSchemaFile(String fileName, Map<String, Object> schemaJsonMap)
+      throws IOException {
     Gson gson = new Gson();
     try (Writer writer =
-        new OutputStreamWriter(new FileOutputStream(SCHEMA_FILE), StandardCharsets.UTF_8)) {
+        new OutputStreamWriter(new FileOutputStream(fileName), StandardCharsets.UTF_8)) {
```

Comment:
My IntelliJ suggested as follows:
```suggestion
        new OutputStreamWriter(Files.newOutputStream(Paths.get(fileName)), StandardCharsets.UTF_8)) {
```

Reply:
Declaring the `FileOutputStream` in a separate variable so that the try-with-resource can close it solved the warning so I did that. 
Cf. https://github.com/scalar-labs/scalardb/pull/648/commits/729a771bb3060c35c158c10eb380fe9d677e718c